src/website/src/app/documentation/get-started/get-started.component.html

```diff
@@ -150,21 +150,30 @@ <h4 id="step6-update">Step 6: Update Clarity</h4>
     <p>If you want to update to a prerelease version, you can do so by running:</p>
     <clr-code-snippet
             clrLanguage="bash"
-            [clrCode]="'ng update @angular/core@next @angular/cli@next
-ng update @clr/angular@next'"
+            [clrCode]="'ng update @angular/core@next @angular/cli@next ng update @clr/angular@next'"
```

Comment:
Revert this it should be two different lines because you don't want to run them both at the same time.